### ISSUE TITLE
Quickstart fails - main.Post does not exist in the current database

### ISSUE THREAD

**Issue by alechp** [prisma/prisma-client-go#478]
### Quickstart document fails
Continuation of https://github.com/prisma/prisma-client-go/issues/475

This exact error message has been replicated by @trading-peter (Linux user, Go programmer) and myself (Mac, currently using JS version of Prisma in production) in fresh environments multiple times. 

```
panic: pql error: Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: Some(KnownError { message: "The table `main.Post` does not exist in the current database.", meta: Object({"table": String("main.Post")}), error_code: "P2021" }), kind: TableDoesNotExist { table: "main.Post" } })

goroutine 1 [running]:
main.main()
	/Users/alechp/Code/roundtabletg/prisma-go/demo/main.go:12 +0x4c
exit status 2
```

### Step-by-step quickstart
PDF contains step-by-step execution of shell commands and screenshots of terminal:
[prisma-client-go Quickstart.pdf](https://github.com/prisma/prisma-client-go/files/6501722/prisma-client-go.Quickstart.pdf)

**Comment by steebchen** [comment]
Thanks, I just tried it with a fresh install and isolated from my own thing, and it's indeed not working. Sorry for closing the other issue too early. I'll look into it.

**Comment by steebchen** [comment]
btw, the error you're getting when first running the go client generate command is related to Go1.16 and our fix is tracked in #452

**Comment by steebchen** [comment]
It looks like there is a bug related to a relative path issue. That means there was a dev.db file in prisma/ from `db push` and in the root directory from the client.

Please move the prisma.schema file into the root, run `db push` again and it should work.

For now, I have adapted the quickstart so that the prisma schema is created in the root.

I will close this and will put further discussion into a more clear issue for planning at #482.

**Comment by alechp** [comment]
@steebchen That fixed it! Thank you

![Screen Shot 2021-05-18 at 12 43 22 PM](https://user-images.githubusercontent.com/3058939/118713327-a390eb00-b7d6-11eb-8f37-1812a6f80d96.png)

**Comment by alechp** [comment]
For anyone using this fix that gets stuck, when you run `go run github.com/prisma/prisma-client-go db push` from the root directory (eg. `/demo/`) it's still expecting the `db` directory to be populated. You can solve this by adding a symlink for db directory.

Initially thought schema.prisma might need be the problem: 
![Screen Shot 2021-05-18 at 12 48 15 PM](https://user-images.githubusercontent.com/3058939/118715276-4c8c1580-b7d8-11eb-90a5-21bf9e824006.png)

but so far it seems like just `db` directory needs to be symlinked:

![Screen Shot 2021-05-18 at 12 55 59 PM](https://user-images.githubusercontent.com/3058939/118715377-67f72080-b7d8-11eb-83bb-47117424b442.png)

TLDR (from root dir):
```
ln -s db ./prisma/db
go run github.com/prisma/prisma-client-go db push
go run .
```

**Comment by steebchen** [comment]
You shouldn't need any symlink, just moving the schema file into the root, ideally deleting the whole `prisma` directory, and then re-running `generate` should do the trick.

**Comment by alechp** [comment]
@steebchen You're right. I forgot to update main.go to revert the previous change from `demo/prisma/db` to `demo/db`

Works with `prisma/` dir deleted:
![Screen Shot 2021-05-18 at 1 55 01 PM](https://user-images.githubusercontent.com/3058939/118722144-a55fac00-b7e0-11eb-9153-5cb3cef24fc0.png)

Updated import statement to reflect (ironically, reverting to where we started):
```
package main

import (
    "context"
    "encoding/json"
    "fmt"
    "demo/db"
)
```
